package-lock.json

```diff
@@ -0,0 +1,1878 @@
+{
```

Comment:
We should not submit `package-lock.json` for this repository and should exclude it in .gitignore after deleting (same for yarn lock).

The reason is that package-lock.json will only mask problems that upstream folks will have because depending on a library does not enforce that library's lockfile.